doc/getting-started/acrn-sanity-test.rst

```diff
@@ -0,0 +1,352 @@
+ACRN Sanity Test
+################
+
+The document describes the daily sanity test for ACRN by validation team, which 
+includes a set of system level core tests of basic functionality to demonstrate 
+ACRN hypervisor’s availability. User could manually run these after setup ACRN. 
+
+ACRN Sanity Test Cases
+######################
+
+Hypervisor
+##########
+
+Introduction
+************
+
+ST_HV_Bootloader_LaaG_VUEFI_Default
+**********************************
+
+* Description
+
+The test case focus on testing boot up a LaaG with VUEFI and default launch_uos.sh.
+
+* Prerequisites
+
+Service OS boots up normally
+
+* Test Steps
+
+1. Launch SOS
+#. Use ``acrn-dm`` to launch clearlinux using the pre-installed default ``launch_uos.sh`` script
+
+  .. code-block:: none
+
+     $ sudo /usr/share/acrn/samples/nuc/launch_uos.sh
+
+* Expected Result
+
+LaaG boot up normally
+
+ST_HV_CPU_LaaG_Partition_Check
```

Comment:
Is this test case still relevant? The number of CPU cores assigned to a VM is no longer handled by `acrn-dm` (via the `launch_uos.sh` script)?